### PR TITLE
Adds info to README

### PR DESCRIPTION
**Pull request by jgreben**
Here is some initial information for the README. I made an assumption about deployment using travis ci which may or may not be true so please confirm and provide me with the right information.

Also, if we are planning to have some kind if database backing this app that will need creation and initialization, we can add that information to the README as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,60 @@
...
+    $ rake rubocop
+
+## Deployment
+
+Deployment of the application will be handled using the DLSS capistrano gem. To deploy the app to the development application 
+server run:
+    
+    $ cap dev deploy
+
+Deployment of the production application will be handled via TravisCI
```
Comment (cbeer) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,60 @@
-# README
+# MyLibrary
```
Comment (cbeer) [inline README.md]: I think there are a bunch of badges we put on our other repositories that wee might want to insert here as well.

Comment (camillevilla) [reply README.md]:
```suggestion
# MyLibrary

[![Test Coverage](https://api.codeclimate.com/v1/badges/a8f1c5dab3a53ffba586/test_coverage)](https://codeclimate.com/github/sul-dlss/mylibrary/test_coverage)
```
^-- Here's the codeclimate one

Comment (camillevilla) [reply README.md]: Aaaand Travis:
```
[![Build Status](https://travis-ci.org/sul-dlss/mylibrary.svg?branch=master)](https://travis-ci.org/sul-dlss/mylibrary)
```

**Pull request merged**